index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="./styles.css">
+    <script src="https://kit.fontawesome.com/59ba0ac1be.js" crossorigin="anonymous"></script>
+</head>
+<body>
+  <header>
+    <a href="./index.html">
+    <img src="./images/logo.png" alt="Company logo" title="company logo" class="img_class">
+    </a>
+    <nav>
+      <ul>
+        <li><a href="./portfolio.html">Portfolio</a></li>
+        <li><a href="./about.html">About</a></li>
+        <li><a href="./contact.html">Contact</a></li>
+      </ul>
+    </nav>
+    <i class="fa-sharp fa-solid fa-bars"></i>
+  </header>
+  <section id="introduction">
```

Comment:
- [ ] Kindly consider using `flex` to align the content in this section the reason is following:
  -  This gives us a lot of options to align all our elements, plus, it's responsive.